### DISCUSSION TITLE
Change name - Round 1

### DISCUSSION THREAD

**Discussion by 0xMRTT** [GradienceTeam/Gradience#113] [Polls]
Recently, a member of GNOME asked us to change the name because Adwaita Manager look like an official app.

So, we found some names.

**Comment by tseli0s** [comment]
Customizer also sounds like a GNOME App. We're gonna need time.

**Reply by 0xMRTT** [reply]
The name change will be in the v0.2.0

**Comment by 0xMRTT** [comment]
@AndroGR what do you want to do for AdwCustomizer? Code, design, community, i18n, just participate in discussions and issues, other ?

**Comment by ArtyIF** [comment]
How about Gradience? [A real word, apparently](https://en.wiktionary.org/wiki/gradience).

**Reply by daudix** [reply]
Sounds very good!

**Comment by 0xMRTT** [comment]
Good one. I think that we can change for Gradience

**Comment by daudix** [comment]
Spectrum?

**Reply by 0xMRTT** [reply]
I prefer Gradience

**Reply by 0xMRTT** [reply]
But good idea
